frontend/targets.md

```diff
@@ -1,2 +1,26 @@
 # Targets
+This section contains the list of the vulnerable machines you can attack. They are ranked by the following difficulties
+
+- Beginner
+- Basic
+- Intermediate
+- Advanced
+- Expert
+- Guru
+
+![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/1bdb5d44-6540-4718-9eda-83cc6cdfdc3b/8076e0e8-ca4e-43e0-a2f7-445a54124aaa/Untitled.png)
+
+### Spawning a Target
+To spawn a target, click on the and click on the settings icon and click on spawn private instance
+
+<aside>
+💡 Note: Private Instances are for ==PAID== users only
+
+</aside>
+
+![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/1bdb5d44-6540-4718-9eda-83cc6cdfdc3b/aea55257-446c-4000-915b-86cf0e9bd003/Untitled.png)
+
+
+
+
```

Comment:
there is an extension for vscode called Trailing Spaces, install it so that there are no empty lines at the end of the file and end of lines